share/spice/symbolab/symbolab.css

```diff
@@ -0,0 +1,4 @@
+.zci--symbolab .c-base__title {
+    overflow: visible;
+    margin: 0.8em 0;
+}
```

Comment:
it looks like we need to reset the `top` for `sup` elements inside our IA's div as our global CSS modified the `top` which is causing rendering problems for fractions:

```css
.zci--symbolab sup {
    top: 0;
}
```